domains/averagepc.json

```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "FaultyFaulty-glitch",
+           "email": "",
+           "discord": "1102928631191322675",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.AJTHGS-kVUpJ5p8utWujGCHEbL03DaN-9ffmEHvO2oJzYbgA2LWN8aRHnbI4-iMorf891AUESaTLi8eRJzGZ92PC2qp2ujFiJ90Xyd-Bff9DJXTyhTkETwurrd_PTJw4cUx-FbdbwM_Zhw3eLKwU2CE3MIbjY-EJ1dEHGJ14hy4SlpGAvlEedF1iO2laMlL2EtRHJKdTn8r2FVlRrSo_ZEoZ3rkNTkcTIPiHXDBuUMwKteQptzbcshBup4PUPyugQt_Zrw1NwwO9qP5vKivtn72AX4V6J66KOCPgDS_QaNKwSQhLUGjXcXQUXzrHOA9t0xgo5H8GSdpuFYhcDvQvvg.kbbRkAhLCaspsL2YU5eoTw.kZZRPh2GaxWmQn08m_cJsr789ZOyuYxF1ghnny6tbeJQcrZGb9DWv-7jF3GolWmhPJpCu7TJ_zvJsvKDFtplsiqJrSUkRPMrbbwAxKN1OC5u1d4ADy6WtAUApaw1NcQJ.0ip2XFGqYoxzSiYRH1Dz5g"
+        },
+    
+        "record": {
+            "TXT": "google-site-verification=rgnl_a1dm87n16yradqi42uiz61mn_3es5g_u1tor-k"
```

Comment:
Ehh dunno if we support this. (If it was along with like a A record then I would merge it, but like this not really)

Reply:
I do remember seeing a lot of people using google sites. Could you try and see if it works?

Reply:
Thats just a TXT, it doesn't point to any of google's servers.
Though im not sure how google sites's process goes so probably soon I'll try.

Reply:
> Thats just a TXT, it doesn't point to any of google's servers. Though im not sure how google sites's process goes so probably soon I'll try.

This is what google says
![image](https://github.com/user-attachments/assets/a954ce8a-8b3e-41e2-a30c-3f83dff9d2e8)